src/main/io/gps.c

```diff
@@ -410,6 +411,12 @@ void gpsInitNmea(void)
                gpsData.state_position++;
            } else if (gpsData.state_position < 2) {
                serialPrint(gpsPort, "$PSRF103,00,6,00,0*23\r\n");
+                // special initialization for NMEA ATGM336 and similar GPS recivers - should be done only once
+               if (!atgmRestartDone) {
```

Comment:
Variable calls out the atgm, but this should be a generic NMEA function. Suggest renaming to `nmeaRestartDone`

Reply:
sure, I can change the name, but the command will restart only atgm chips, so it's not 100% NMEA relevant...

Reply:
ok, if this is ATGM only it makes sense.